operator/controllers/internal/csiscaleoperator/csiscaleoperator.go

```diff
@@ -0,0 +1,168 @@
+package csiscaleoperator
+
+import (
+	"k8s.io/apimachinery/pkg/labels"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	csiv1 "github.com/IBM/ibm-spectrum-scale-csi/api/v1"
```

Comment:
change to 
        csiv1 "github.com/IBM/ibm-spectrum-scale-csi/operator/api/v1"
	"github.com/IBM/ibm-spectrum-scale-csi/operator/controllers/config"

Reply:
Done